src/ui.rs

```diff
@@ -54,22 +52,22 @@ impl Widget for &Thok {
                                 ((area.height as f64 - prompt_occupied_lines as f64) / 2.0) as u16,
                             ),
                             Constraint::Length(time_left_lines),
-                            Constraint::Length(prompt_occupied_lines),
+                            Constraint::Length(3),
```

Comment:
should this be at most 3? If a prompt only takes up 1 line we shouldn't give it 3

Reply:
I don't think it actually matters since those lines wouldn't be visible would they? Its just empty space.